.circleci/unittest/linux/scripts/environment.yml

```diff
@@ -1,13 +1,14 @@
 channels:
   - pytorch
   - defaults
+  - conda-forge
```

Comment:
cc @malfet as previously you've tried to phase out conda-forge from TorchVision.

Reply:
There are two issues I'm slightly concerned about:
  - Would it mean that we are testing something different then what we ship?
  - This might unintentionally pull in different dependencies, that we think exist in conda, but they are not.

If this is only needed for testing, we can build and host `libjpeg-turbo` for all the platforms we care about in `pytorch` or `pytorch-nightly` channels

Reply:
Thanks for your feedback Nikita

> Would it mean that we are testing something different then what we ship?

Yes, but only until https://github.com/pytorch/vision/pull/5951 is merged. https://github.com/pytorch/vision/pull/5951 's goal is to ship torchvision with libjpeg turbo.

It's also not as bad as it sounds: it's fair at this point to assume that libjpeg-turbo's `libjpeg.h` is compatible with that of libjpeg. In terms of decoding results, they're both jpeg-compliant as well.

> If this is only needed for testing, we can build and host libjpeg-turbo for all the platforms we care about in pytorch or pytorch-nightly channels

Ultimately we don't want this just for testing: we also want to ship torchvision with libjpeg in https://github.com/pytorch/vision/pull/5951 . But if we're happy to host libjpeg-turbo in the `pytorch` channel, then this might make all this much easier. WDYT @malfet